libs/devices/harmony/routes/src/lib/screens/manage-files/harmony-manage-files.screen.tsx

```diff
@@ -3,37 +3,112 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-import { FunctionComponent } from "react"
+import { FunctionComponent, useCallback, useMemo, useState } from "react"
 import { DashboardHeaderTitle } from "app-routing/feature"
 import { defineMessages, formatMessage } from "app-localize/utils"
+import {
+  FileManagerFile,
+  ManageFiles,
+  ManageFilesLoadingState,
+} from "devices/common/ui"
+import { HarmonyManageFilesTableSection } from "./harmony-manage-files-table-section"
 import { useHarmonyManageFiles } from "./use-harmony-manage-files"
+import { FileCategoryId } from "./harmony-manage-files.types"
+import { useActiveDeviceQuery } from "devices/common/feature"
+import { Harmony } from "devices/harmony/models"
 
 const messages = defineMessages({
   pageTitle: {
     id: "page.manageFiles.title",
   },
+  summaryHeader: {
+    id: "manageFiles.summary.harmonyHeader",
+  },
 })
 
 export const HarmonyManageFilesScreen: FunctionComponent = () => {
+  const { data: activeDevice } = useActiveDeviceQuery<Harmony>()
   const {
-    isError,
     isLoading,
     segments,
     categories,
     categoryFileMap,
     freeSpaceBytes,
     usedSpaceBytes,
     otherSpaceBytes,
-  } = useHarmonyManageFiles()
+  } = useHarmonyManageFiles(activeDevice)
 
-  console.log("isError, isLoading, data", isError, isLoading, {
-    segments,
-    categories,
-    categoryFileMap,
-    freeSpaceBytes,
-    usedSpaceBytes,
-    otherSpaceBytes,
-  })
+  const [activeCategoryId, setActiveCategoryId] = useState<
+    FileCategoryId | undefined
+  >(categories[0]?.id)
+
+  const [selectedIds, setSelectedIds] = useState<Set<string>>(() => new Set())
+
+  const activeFileMap = useMemo(
+    () => (activeCategoryId ? (categoryFileMap[activeCategoryId] ?? {}) : {}),
+    [activeCategoryId, categoryFileMap]
+  )
+
+  const selectedFiles: FileManagerFile[] = useMemo(() => {
+    const out: FileManagerFile[] = []
+    selectedIds.forEach((id) => {
+      const f = activeFileMap[id]
```

Comment:
Try to avoid using single-letter names, as they're not self-descriptive and reduce maintainability.